perun-base/src/main/java/cz/metacentrum/perun/core/api/RoleManagementRules.java

```diff
@@ -23,6 +23,10 @@
  *            Example entry: key: User; value: user_id
  * assignedObjects is a map of objects which can be assigned with the role. Key is a object name and value is mapping to the database.
  *            Example entry: key: Resource; value: resource_id
+ * assignmentCheck is a list of maps defining which of the assigned objects should be checked for being critical (requiring MFA) when setting the role.
```

Comment:
```suggestion
 * assignmentCheck is a list of maps defining which of the complementary objects should be checked for being critical (requiring MFA) when setting the role.
```
Wouldn't this be more consistent with the Javadocs?

Reply:
i think not - complementary object in javadocs is referred to as the object the role is assigned to (resourceadmin -> resource), while assigned objects refer to all the related objects (vo, resource, facility), which in perun-roles and the line above is called `assignedObjects`